src/main/java/Lotto/domain/EqualLottoCntInfo.java

```diff
@@ -0,0 +1,49 @@
+package Lotto.domain;
+
+import Lotto.exception.NotFoundRankException;
+
+import java.util.stream.Stream;
+
+public enum EqualLottoCntInfo {
+
+    FIRST(6, 2000000000),
+    SECOND(5, 1500000),
+    THIRD(4, 50000),
+    FOURTH(3, 5000),
+    FIFTH(2, 0),
+    SIX(1, 0),
+    NONE(0, 0);
+
+    private static final int MIN_COUNT = 3;
+
+    private final int equalCnt;
+    private final int winningMoney;
+
+    EqualLottoCntInfo(int equalCnt, int winningMoney) {
+        this.equalCnt = equalCnt;
+        this.winningMoney = winningMoney;
+    }
+
+    public static EqualLottoCntInfo findEqualLottoInfo(int equalCnt) {
+        return Stream.of(values())
+                .filter(WinningRankInfo -> (WinningRankInfo.equalCnt == (equalCnt)))
```

Comment:
```suggestion
                .filter(winningRankInfo -> (winningRankInfo.equalCnt == (equalCnt)))
```
지역변수이기 때문에 수정이 필요해보여요 :)

Reply:
꼼꼼하십니다 ㄷㄷㄷ